src/lib/Timer.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState, useEffect } from 'react';
+import './styles.css';
+
+function Timer({ stopTimer }) {
+  const [minute, setMinute] = useState(0);
+  const [second, setSecond] = useState(0);
+  useEffect(() => {
+    let intervalId;
+
+    if (!stopTimer) {
+      intervalId = setInterval(() => {
+        if (second >= 59) {
+          setMinute((prevMinute) => prevMinute + 1);
+          setSecond(0);
+        } else {
+          setSecond((prevSecond) => prevSecond + 1);
+        }
+      }, 1000);
+    }
+
+    return () => {
+      clearInterval(intervalId);
+    };
+  }, [stopTimer, second]);
+
+  return (
+    <div className="timer-container">
+      {stopTimer ? (
+        <div>
+          <h3 className="total-time">
+            Total time taken:
+            {`${minute}:${second} seconds`}
```

Comment:
1. add a space after `Total time taken:`
2. I don't think we need the unit here. 

<img width="470" alt="image" src="https://github.com/wingkwong/react-quiz-component/assets/35857179/9476b658-c2c4-4b28-b5a0-d74bd215e669">